.github/workflows/ci.yml

```
@@ -0,0 +1,107 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  testsuite:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version: ['7.4', '8.0']
+        prefer-lowest: ['']
+        include:
+          - php-version: '7.4'
+            prefer-lowest: 'prefer-lowest'
+
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        fetch-depth: 1
+
+    - name: Setup PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: ${{ matrix.php-version }}
+        extensions: mbstring, intl
+        coverage: pcov
+
+    - name: Get composer cache directory
+      id: composer-cache
+      run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+    - name: Get date part for cache key
+      id: key-date
+      run: echo "::set-output name=date::$(date +'%Y-%m')"
+
+    - name: Cache composer dependencies
+      uses: actions/cache@v1
+      with:
+        path: ${{ steps.composer-cache.outputs.dir }}
+        key: ${{ runner.os }}-composer-${{ steps.key-date.outputs.date }}-${{ hashFiles('composer.json') }}-${{ matrix.prefer-lowest }}
+
+    - name: Composer Install
+      run: |
+        if ${{ matrix.prefer-lowest == 'prefer-lowest' }}; then
+          make install-dev-lowest
+        else
+          make install-dev
+        fi
+
+    - name: Configure PHPUnit matcher
+      if: matrix.php-version == '7.4'
+      uses: mheap/phpunit-matcher-action@master
```

```suggestion
      uses: mheap/phpunit-matcher-action@v1
```

Pinning to master seems dangerous.